install/playbooks/roles/ldap/vars/main.yml

```diff
@@ -5,3 +5,19 @@ users_defaults:
   uid_start: 1001
   gid_start: 1001
   shell: /bin/dash
+
+
+packages:
+  install:
+    - acl
+    - slapd
+    - python-ldap
+    - python-passlib
+    - ldapscripts
+    - libpam-pwquality
+    - ldapvi
+    - nscd
+  remove:
+    - libpam-cracklib
+    - cracklib-runtime
+    - libcrack2
```

Comment:
`libcrack2` seems to be a dependency of `libpam-pwquality` via `libpwquality1`, better not try to remove it. 

So `cracklib-runtime` gets removed. it was installed before. It is not used?

Do you know by what dependency `libpam-cracklib` got installed? I don't have it here, but it is a build from before the changes tested by the CI.

Reply:
The LDAP server fail to start if both libraries are used. Maybe a recent update in Debian that pushed libpam-cracklib in the default packages to install?.
This is minor, and more related to #258.

Reply:
I followed your advice on this one, done.